docs/documentation/development/bindings/faq.md

```diff
@@ -36,3 +36,9 @@ Here is a list of frequently asked questions around the development of bindings.
 5. _When creating a Thing through my ThingHanderFactory, does it exactly have to have the channels that are defined in the thing type description?_
  
     It must at least have the channels that are defined in its thing type (and they are already automatically added by the super() implementation). Nonetheless, you are free to add any number of additional channels to the thing.
+
+### Presenting device data to a user
```

Comment:
We usally use capitals in headers.
Apart from this, this header is a bit fuzzy as it sounds as if it is about UIs.
Better change it to something like "State Updates of Channels".